app/main/forms.py

```diff
@@ -2692,7 +2693,7 @@ class ChangeSecurityKeyNameForm(StripWhitespaceForm):
     security_key_name = GovukTextInputField(
         "Name of key",
         validators=[
-            DataRequired(message="Cannot be empty"),
+            DataRequired(message="enter a name for this key"),
```

Comment:
```suggestion
            DataRequired(message="Enter a name for this key"),
```

And we probably should say Yubikey instead of just key? @karlchillmaid